source/Generic/GamesSizeCalculator/GamesSizeCalculatorSettingsView.xaml

```diff
@@ -6,37 +6,84 @@
              xmlns:pcmd="clr-namespace:PluginsCommon.Commands"
              mc:Ignorable="d"
              d:DesignHeight="400" d:DesignWidth="600">
-    <StackPanel Margin="20">
-        <TextBlock VerticalAlignment="Center" Margin="0,0,0,0">
-                <Hyperlink NavigateUri="https://github.com/darklinkpower/PlayniteExtensionsCollection/wiki/Games-Size-Calculator"
-                           Command="{x:Static pcmd:GlobalCommands.NavigateUrlCommand}"
-                           CommandParameter="{Binding NavigateUri, RelativeSource={RelativeSource Self}}">
-                    <Run Text="{DynamicResource LOCGame_Sizes_Calculator_SettingsHelpLabel}" />
-                </Hyperlink>
-        </TextBlock>
-        <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsCalculateOnGameImportLabel}"
-                  Margin="0,10,0,0"
-                  IsChecked="{Binding Settings.CalculateNewGamesOnLibraryUpdate}"/>
-        <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsCalculateOnGameModifiedLabel}"
-                  Margin="0,10,0,0"
-                  ToolTip="{DynamicResource LOCGame_Sizes_Calculator_SettingsCalculateOnGameModifiedTooltip}"
-                  IsChecked="{Binding Settings.CalculateModifiedGamesOnLibraryUpdate}"/>
-        <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsCalculateModifiedGamesOnLibraryUpdateLabel}"
-                  Margin="0,10,0,0"
-                  IsChecked="{Binding Settings.GetUninstalledGameSizeFromSteam}"/>
-        <StackPanel Margin="40,0,0,0"
-                    IsEnabled="{Binding Settings.GetUninstalledGameSizeFromSteam}">
-            <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsGetUninstalledGameSizeFromSteamLabel}"
-                      Margin="0,10,0,0"
-                      IsChecked="{Binding Settings.GetSizeFromSteamNonSteamGames}"/>
-            <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsIncludeDlcInSteamCalculationLabel}"
-                      Margin="0,10,0,0"
-                      IsChecked="{Binding Settings.IncludeDlcInSteamCalculation}"/>
-            <CheckBox Content="{DynamicResource LOCGame_Sizes_Calculator_SettingsIncludeOptionalInSteamCalculationLabel}"
-                      Margin="0,10,0,0"
-                      ToolTip="{DynamicResource LOCGame_Sizes_Calculator_SettingsIncludeOptionalInSteamCalculationTooltip}"
-                      IsChecked="{Binding Settings.IncludeOptionalInSteamCalculation}"/>
-        </StackPanel>
+    <TabControl ScrollViewer.VerticalScrollBarVisibility="Disabled" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch">
```

Comment:
Use a Panel as the main control here please. Either `Grid` or `StackPanel` with a margin of `20` for consistency and place the `TabControl as a child`. Also what is the purpose of `ScrollViewer.VerticalScrollBarVisibility` and the ContenAlignment properties?

Reply:
Why an extra wrapping element rather than a Margin="20" for the TabControl?

The properties are for the second tab, I'll move them there.